bigtop-packages/src/charm/spark/layer-spark/reactive/spark.py

```diff
@@ -38,13 +38,15 @@ def report_status():
                            'yarn execution mode not available')
         return
 
-    if mode == 'standalone' and is_state('leadership.is_leader'):
+    if mode == 'standalone' and is_state('zookeeper.ready'):
+        mode = mode + " - HA"
+    elif mode == 'standalone' and is_state('leadership.is_leader'):
```

Comment:
Yes there is the case where zookeeper ready and is leader is set. This happens when the leaderelection layer has elected the unit to be a leader but at the same time the master spark unit is to be selected via zookeeper. In this state the leader election layer should be ignored as the master will be decided by zookeeper.